Tests/TuistLoaderTests/Mappers/ProjectDescriptionHelpersLocationMapperTests.swift

```diff
@@ -0,0 +1,154 @@
+import ProjectDescription
+import TSCBasic
+import TuistCore
+import TuistSupport
+import XCTest
+
+@testable import TuistCoreTesting
+@testable import TuistLoader
+@testable import TuistSupportTesting
+
+final class ProjectDescriptionHelpersLocationMapperTests: TuistUnitTestCase {
+    private let fakeRoot = AbsolutePath("/Root")
+    private var rootDirectoryLocator: MockRootDirectoryLocator!
+    private var derivedDirectory: AbsolutePath!
+
+    override func setUp() {
+        super.setUp()
+        rootDirectoryLocator = MockRootDirectoryLocator()
+        rootDirectoryLocator.locateStub = fakeRoot
```

Comment:
how about we generate `fakeRoot` with `temporaryPath()` function? And change `fakeRoot` to `private var fakeRoot: AbsolutePath!`